src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx

```diff
@@ -53,7 +53,7 @@ In contrast, with "tumbling" or "cascading" windows, the windows do not overlap.
 
 ## When to use sliding windows [#when-to-use]
 
-Sliding windows are helpful when you need to smooth out "spiky" charts. One common use case is to use sliding windows to smooth line graphs that have a lot of variation over short periods of time in cases where the rolling aggregate (for example a rolling mean) is more important than aggregates from narrow windows of time.
+Sliding windows are helpful when you need to smooth out "spiky" charts. It's common to use sliding windows to smooth line graphs that have a lot of variation over short periods of time in cases where the rolling aggregate (for example a rolling mean) is more important than aggregates from narrow windows of time.
```

Comment:
"It's common..." --> "You can use sliding windows to smooth spiky line graphs that have many variations over short periods of time. Especially use this when the rolling aggregate of the data is more important to you than etcetc"